functest/logicrunner_test.go

```diff
@@ -381,9 +385,19 @@ func (r *Two) Value() (int, error) {
 	resp := callMethod(t, obj, "Hello")
 	require.Empty(t, resp.Error)
 
-	resp = callMethod(t, obj, "Value")
-	require.Empty(t, resp.Error)
-	require.Equal(t, float64(644), resp.ExtractedReply)
+	testPassed := false
+	for i := 0; i < 25; i++ {
+		resp = callMethod(t, obj, "Value")
+		require.Empty(t, resp.Error)
+		fmt.Println(">>>> resp.ExtractedReply: ", resp.ExtractedReply)
+		if float64(644) == resp.ExtractedReply {
+			testPassed = true
+			break
+		}
+		time.Sleep(1000 * time.Millisecond)
```

Comment:
equal to `time.Second`, but it's better to create "exponential waiting", IMHO

Reply:
bug situations became only on pulse changes, so i just add time for waiting a few pulses